src/Symfony/Component/HttpKernel/Client.php

```diff
@@ -48,26 +48,6 @@ public function __construct(HttpKernelInterface $kernel, array $server = array()
     }
 
     /**
-     * {@inheritdoc}
-     *
-     * @return Request|null A Request instance
-     */
-    public function getRequest()
```

Comment:
These 2 methods are intended in order to get correct autocompletion in IDE's. (the base client can only typehint `object`, while the HttpKernel client knows that object is a HttpFoundation request/response).